components/focus-trap/focus-trap.js

```diff
@@ -29,7 +29,7 @@ class FocusTrap extends FocusMixin(LitElement) {
 				display: inline-block;
 			}
 			:host([hidden]) {
-				display: hidden;
+				display: none;
```

Comment:
`hidden` is not a supported value for `display`. I'll confirm that no where was using `d2l-focus-trap hidden` as this would suddenly cause previously visible cases to now not be.

Reply:
I have not found any cases where `hidden` is used on `d2l-focus-trap` so this change should be safe.